strawberry_django/filters.py

```diff
@@ -169,8 +169,7 @@ def _resolve_global_id(value: Any):
 
 
 def build_filter_kwargs(
-    filters: WithStrawberryObjectDefinition,
-    path="",
+    filters: WithStrawberryObjectDefinition, path="", is_lookup=False
```

Comment:
question: why do we need to pass `is_lookup` here? Isn't it just a matter of doing `is_lookup = isinstance(field_value, FilterLookup)` at the beggining of the function?

Reply:
it looses that information because of the cast, so this is the only place to know

Reply:
do you have a better idea?